README.md

```diff
@@ -26,11 +26,11 @@ This is a VS Code extension which enables the interactive debugging of DFDL sche
 
 ## Installation
 
-Until the extension is available in the [VS Code Extension Marketplace](https://marketplace.visualstudio.com/vscode), please download the latest `.vsix` file from the [releases page](https://github.com/jw3/example-daffodil-vscode/releases). Then install it by either:
+Until the extension is available in the [VS Code Extension Marketplace](https://marketplace.visualstudio.com/vscode), please download the latest `.vsix` file from the [releases page](https://github.com/apache/daffodil-vscode/releases). Then install it by either:
   * using the "Extensions: Install from VSIX" command from within VS Code (open the Command Palette with Shift-Command-P, then type `vsix` to bring up the command and pointing it at the downloaded `vsix` file); or
   * on the command-line via `code --install-extension <path-to-downloaded-vsix>`.
 
 
 ## Usage
 
-Please refer to the Wiki page at https://github.com/jw3/example-daffodil-vscode/wiki
+Please refer to the Wiki page at https://github.com/apache/daffodil-vscode/wiki
```

Comment:
No where in the README are there build dependencies or instructions. Apache is very source code focused, so instructions should first and foremost be about how to build things from source. It's fine to also provide convenience binaries, whether they be from downloads.apache.org or via the marketplace, but that's importtation information that should be in this README. It doesn't need to be part of this PR, but a new issue should be opened to add that information.